libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFPushNotificationManager.m

```diff
@@ -103,6 +107,7 @@ + (SFPushNotificationManager *) sharedInstance
     dispatch_once(&pred, ^{
         mgr = [[super allocWithZone:nil] init];
     });
+    [[SalesforceSDKManager sharedManager] registerAppFeature:kSFAppFeaturePushNotifications];
```

Comment:
I think this should be inside the `dispatch_once`, since we don't need it more than once.